addons/medical/functions/fnc_isMedic.sqf

```diff
@@ -19,7 +19,7 @@
 
 params ["_unit", ["_medicN", 1]];
 
-private _class = _unit getVariable [QGVAR(medicClass), getNumber (configFile >> "CfgVehicles" >> typeOf _unit >> "attendant")];
+private _class = _unit getVariable [QGVAR(medicClass), _unit getUnitTrait "medic"];
```

Comment:
I think that should be done in medical-rewrite, as we can afford to change things there that would otherwise be backwards-incompatible.